### DISCUSSION TITLE
error when uploading file

### DISCUSSION THREAD

**Discussion by sagihgithub** [community/community#91463] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi
A week ago I was able to load files to my repository with no problem.
Since yesterday when ever I'm trying to upload any kind of file I'm getting the following error:
'Something went really wrong, and we can’t process that file.'
Also, I'm getting 'Failed to load resource: net::ERR_CONNECTION_TIMED_OUT' error in developer tool panel.
I'm talking about a 100kb file size
I've tried from different browsers (chrome, edge) with no success
Appreciate any help

**Comment by jimsideout** [comment]
Have you solved this @sagihgithub ?

**Comment by saifeddineelhanoune** [comment]
GitHub has a file size limit of 10 MB for individual files If the file you are trying to upload is larger than this you will get an error message However you mentioned that the file size is only 100 KB so this is likely not the issue

**Reply by MubarmajM** [reply]
The error massage would be "Yowza, that's too large" or something, not "Something went really work"

**Comment by saifeddineelhanoune** [comment]
"net::ERR_CONNECTION_TIMED_OUT" error suggests there might be a network issue This could be due to a VPN or firewall settings that are blocking the upload

**Comment by saifeddineelhanoune** [comment]
Browser extensions can sometimes interfere with web applications you should try bro disabling extensions or using an incognito/private browsing window to see if the issue persists

**Reply by jimsideout** [reply]
I'm attempting to upload a file to github releases. Doing it directly on the website gives me a response of "Something went really wrong, and we can’t process that file".

**Reply by jimsideout** [reply]
I've tried different browsers, incognito, etc... same result.

Using command line I get "⨯ 001076D901000000:error:0A0003FC:SSL routines:ssl3_read_bytes:sslv3 alert bad record mac:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1586:SSL alert number 20"

Sometimes certain files work and other times they don't. There seems to be no pattern.

**Reply by saifeddineelhanoune** [reply]
There could be temporary issues with GitHub's servers sometimes these issues resolve themselves after some time you can check GitHub's status page to see if there are any ongoing issues and ensure again that the file you're trying to upload has the correct permissions some users have reported issues when their browser did not have permission to access the file's location that's all what i know about file uploading family

**Reply by saifeddineelhanoune** [reply]
but you can pass them by gitCLI if you want surely they will push

**Reply by jimsideout** [reply]
CLI gives the sslv3 alert bad record mac error

Website gives "Failed to load resource: net::ERR_SSL_BAD_RECORD_MAC_ALERT" error

**Reply by wezlol** [reply]
I just had the same issue.

I tried uploading code (18).html from chrome downloads, however I had already renamed it to life.html through explorer.exe

uploading life.html gave me a error 400. Then I refreshed explorer.exe (F5) and dragged life.html to upload & it works!

_url: /upload/main_

**Comment by kisahklasik** [comment]
image no resposive profilepicture generator?

**Comment by kisahklasik** [comment]
![IMG_20240312_000801](https://github.com/community/community/assets/135429317/d2f1cc5a-83db-4d10-ba4c-a3def1f4471b)

demo 👉 https://posterwall.netlify.app/

**Comment by Islam-Alshaer** [comment]
Have you solved that problem ? I have the same issue trying to upload a release file

**Comment by wira09** [comment]
Quick Solution

Clear browser cache and cookies - Clear the cache for github.com in all browsers you use
Try incognito/private mode - Test uploading files in private browsing mode
Restart router/modem - Turn it off for 30 seconds and then back on

Technical Solution

Use Git command line as an alternative:
bashgit add filename
git commit -m "Upload file"
git push origin main

Check your internet connection:

Test upload speed at speedtest.net
Try accessing another site to make sure your connection is stable

Disable VPN/Proxy if using
Flush DNS:

Windows: ipconfig /flushdns in Command Prompt
Mac: sudo dscacheutil -flushcache

**Reply by AdrianTP** [reply]
Temporarily changed DNS settings for active network adapter on Windows 11 from DHCP to 8.8.8.8/8.8.4.4, then the upload just magically worked. I use a PiHole as my DNS, so this might be part of the issue. Not sure _why_ but whatever.

**Comment by LyraHarring** [comment]
I faced the same issue, uploading small files (100KB) was failing with the error: "Something went really wrong, and we can’t process that file." In DevTools, I saw [net::ERR_CONNECTION_TIMED_OUT](https://cheapsslweb.com/blog/err-ssl-bad-record-mac-alert-error/). Turned out it was a temporary server-side issue. Clearing the cache and trying in incognito didn't help, but it worked after a while, so it might be best to wait or check with support if it persists.

I hope it helps!

**Comment by AsutoshaNanda** [comment]
I too am having the same issue , is there any solution for this . In my case I am uploading few pngs to my repo. But I checked with pdfs, it is working fine.. so ig some issue with uploading images...

**Comment by Manzi-Elvis** [comment]
Hey there!  

Sounds frustrating, especially since it was working fine last week. Based on what you’re describing, here’s a step-by-step way to troubleshoot:

## 1. Check Your Internet Connection
The `ERR_CONNECTION_TIMED_OUT` usually means your browser couldn’t reach GitHub properly.  

- Make sure your connection is stable.  
- If on Wi-Fi, try switching to a wired connection or a different network.  
- Disable VPNs or proxies temporarily—they can sometimes interfere.

## 2. Clear Browser Cache / Try Incognito
Sometimes cached data causes upload errors.  

- Open a private/incognito window and try uploading again.  
- Clear browser cache/cookies and retry.

## 3. Try Git CLI
Instead of uploading via the web interface, use Git commands:  

```bash
git add <file>
git commit -m "Add file"
git push origin main

**Reply by AsutoshaNanda** [reply]
Hey Manzi, Asutosha this side. I tried following your steps but didn't got any results. Do you have any other solution? In my case it is just saying 'something really wrong ....'.

If you have any solution please respond to this.

Regards.

**Comment by Owensitoow** [comment]
¿Has resuelto esto ?

**Comment by wezlol** [comment]
If you keep receiving mails like i did.

Press the unsubscribe url in your mail and log in

https://github.com/settings/notifications for more settings
